### ISSUE TITLE
Internal links starting with `http` (not `https`) are not replaced while duplicating region contents

### ISSUE THREAD

**Issue by MizukiTemma** [digitalfabrik/integreat-cms#3097]
### Describe the Bug

When you create a new region, you can import contents from an existing region (often from Deutschlandvorlage in the live system). If those contents contain links to contents of the copied region, the region slugs in the links will be replaced with that of the region being newly created.

### Steps to Reproduce

1. Go to any region A and add into a page a link to a content of the region A with prefix `http` into a content.
2. Go to the network management and create a new region with importing contents from the region A
3. Go to the new region
4. See the region slug of the internal link is not replaced

If `http://integreat.app/region_a/de/page_title/` is used in the step 1, `http://integreat.app/new_region/de/page_title/` must appear in the step 4.
If `https://integreat.app/region_a/de/page_title/` is used, `https://integreat.app/new_region/de/page_title/` appears (success).

### Expected Behavior

The region slug must be replaced even if URLs are starting with `http` instead of `https`

### Actual Behavior

The region slug is not replaced.

### Additional Information

<details>
  <summary>Traceback</summary>

  ```
  ```

</details>

**Comment by david-venhoff** [comment]
Would this be solved if we implement #68?

**Comment by dkehne** [comment]
@david-venhoff no, because #68 is just about marking http-links as invalid. the problem here is that http links are not recognized as internal links and so not replaced.

What Would Actually Fix #3097? Add HTTP versions to LINKCHECK_SITE_DOMAINS:

```
  LINKCHECK_SITE_DOMAINS: Final[list[str]] = [
      WEBAPP_URL,                           # https://integreat.app
      WEBAPP_URL.replace("https://", "http://"),  # http://integreat.app
      SHORT_LINKS_URL,
      SHORT_LINKS_URL.replace("https://", "http://"),
  ]

```
This is a ~2 line change in settings.py.

**Comment by jonbulz** [comment]
Probably also related to #3746
